scripts/ci/Xvfb-contour-run.sh

```diff
@@ -35,7 +35,6 @@ $CONTOUR_PREFIX \
         debug "$LOG" \
         display ${DISPLAY} \
         early-exit-threshold 0 \
-        dump-state-at-exit "${DUMP_DIR}" \
```

Comment:
we have some issue with ub 20.04 dump state, but I can not repro it inside vm (on real display and wm), so i disabled it just to check that we can run contour on ub with qt5, since dump is available for notcurses gui if needed